auth/src/ts/main.ts

```diff
@@ -14,16 +14,13 @@
 
 // This sample is TypeScript ported version of https://github.com/firebase/quickstart-js.
 
-/// <reference path="../../../node_modules/firebase/app.d.ts" />
-/// <reference path="../../../node_modules/firebase/firebase.d.ts" />
```

Comment:
this lines is not required.
tsc can lookup these definition files. see `cat node_modules/firebase/package.json | grep types`.

http://qiita.com/vvakame/items/72d22e33632178f7db24